### PR TITLE
UI content updated

### PR DESCRIPTION
**Pull request by stanvir75**

### CODE REVIEW

File: server/views/partials/bottom_nav.ejs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+
+<!-- 
+Author: Multiverse Design
+Date: Nov-10-2021
+FileName : bottom_nav.js -->
+
+
```
Comment (Fasop4) [inline server/views/partials/bottom_nav.ejs]: everything here could be placed on the footer.ejs as the footer is only containing some lines of bootstrap scripts for styling. not necessary to have two

File: server/views/partials/footer.ejs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+  
+ <!-- 
+Author: Multiverse Design
+Date: Nov-10-2021
+FileName : footer.js --> 
+
```
Comment (Fasop4) [inline server/views/partials/footer.ejs]: place her nav footer section

**Review by Fasop4 — APPROVED** [review]
In general is ok. just merge all the information in bottom-nav.ejs into footer, delete the bottom-nav file and delete any declaration of bottom-nav on main index view

File: server/views/partials/footer.ejs (hunk 2 above)
Comment (stanvir75) [reply server/views/partials/footer.ejs]: I have transferred from bottom_nav to footer section

File: server/views/partials/bottom_nav.ejs (hunk 1 above)
Comment (stanvir75) [reply server/views/partials/bottom_nav.ejs]: I have deleted the bottom_nav file

**Pull request merged**